### PR TITLE
Migrate to null safety

### PR DESCRIPTION
**Pull request by TimWhiting**
Hi,

There were no changes needed to support dart's nullsafety other than updating dependencies and the sdk version.

Thanks for the package.

### CODE REVIEW

**Comment by kmorkos** [comment]
Thank you for this! I will try to push this in the next few days.

File: pubspec.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 name: dart_random_choice
 description: A library for generating a random choice from an iterable.
-version: 0.0.2
+version: 0.0.3-nullsafety.0
```
Comment (kmorkos) [inline pubspec.yaml]: Following https://dart.dev/null-safety/migration-guide#package-version, I'd prefer to increment the minor version

```suggestion
version: 0.1.0-nullsafety.0
```

File: pubspec.yaml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 issue_tracker: https://github.com/kmorkos/dart_random_choice/issues
 documentation:
 
 environment:
-  sdk: '>=2.0.0 <3.0.0'
+  sdk: '>=2.12.0-0 <3.0.0'
 
 dev_dependencies:
-  test: ^1.6.1
+  test: ^1.16.0-nullsafety.16
```
Comment (kmorkos) [inline pubspec.yaml]:
```suggestion
  test: ^1.16.2
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+## [0.0.3-nullsafety.0] - 01/26/2021
```
Comment (kmorkos) [inline CHANGELOG.md]:
```suggestion
## [0.1.0-nullsafety.0] - 02/15/2021
```

**Review by kmorkos — CHANGES_REQUESTED** [review]
Let's increment the minor version as per https://dart.dev/null-safety/migration-guide#package-version

**Comment by TimWhiting** [comment]
Should be good to go now.

**Review by kmorkos — APPROVED** [review]

**Comment by kmorkos** [comment]
Published https://pub.dev/packages/dart_random_choice/versions/0.1.0-nullsafety.0. Thank you for your contribution @TimWhiting!

**Pull request merged**